### DISCUSSION TITLE
Freeze after "Check 41/130: Your application has sufficient disk space. Passed" How to diagnose?

### DISCUSSION THREAD

**Discussion by dedfft** [enlightn/enlightn#173] [Q&A]
Hello dear community. I launched Enlightn Pro for the first time and it every time freezes at this check 
"Check 41/130: Your application has sufficient disk space. Passed"
How can I diagnose why it is stuck and what error it is analyzing so it stops?

**Comment by daleran** [comment]
@dedfft  I have the same issue. It hangs after 41. I have attempted to disable 41 and 42 but it still hangs none the less. I have waited 2+ hours and it sill does not continue.
